workflow/workflow.go

```diff
@@ -790,7 +790,7 @@ func DataConverterWithoutDeadlockDetection(c converter.DataConverter) converter.
 
 // DeterministicKeys returns the keys of a map in deterministic (sorted) order. To be used in for
 // loops in workflows for deterministic iteration.
-func DeterministicKeys[K constraints.Ordered, V any](m map[K]V) []K {
+func DeterministicKeys[K cmp.Ordered, V any](m map[K]V) []K {
```

Comment:
Since it's an interface constraint, this seems to be backwards compatible

Reply:
Yes, this is backward compatible.